contracts/test/integration/IntegrationTestPriceBoundPSMUSDC.t.sol

```diff
@@ -136,12 +148,22 @@ contract IntegrationTestPriceBoundPSMUSDCTest is DSTest {
 
     /// @notice PSM is set up correctly and view functions are working
     function testGetMintAmountOut() public {
-        uint256 amountUSDCIn = 100;
+        uint256 amountUSDCIn = 100e18;
+
+        uint256 currentPegPrice = oracle.getCurrentOraclePrice();
+
+        uint256 fee = (amountUSDCIn.mul(1e12) * psm.mintFeeBasisPoints()) /
+            Constants.BASIS_POINTS_GRANULARITY;
+
+        uint256 amountOut = ((amountUSDCIn.mul(1e18) / currentPegPrice)).mul(
+            1e12
+        ) - fee;
```

Comment:
there should be a comment here explaining all the things that are happening and why